docs-ref-conceptual/update-azure-cli.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Update the Azure CLI
```

Comment:
Upgrade Azure CLI

Reply:
Please also change the file name.

Reply:
@dbradish-microsoft I talked to @yungezz offline and we agreed to use `Update` in section name to keep consistent with existing doc.